eip-0021.md

```diff
@@ -0,0 +1,60 @@
+# Genuine tokens verification
+
+* Author: @MrStahlfelge
+* Status: Proposed
+* Created: 31-Aug-2021
+* License: CC0
+* Forking: not needed 
+
+## Description 
+
+This EIP lists the common tokens of value with their unique identifier, 
+so users as well as wallet and explorer applications can verify if a token is genuine. The EIP is meant to be updated regularly when new tokens of value are added.
+
+## Motivation 
+
+Ergo tokens can hold a certain value, best-known examples are the SigUSD and SigRSV tokens in use by the SigmaUSD stablecoin protocol. 
+Tokens can be minted by every user, with a name and description free to choose. This means everyone can mint new tokens named "SigUSD", which bears a problem for 
+end-users to decide if a token they received is genuine or not.
+
+## Ergo tokens background
+
+[See EIP-4](eip-0004.md): Ergo supports custom tokens as first-class citizens. A transaction can create out-of-thin-air tokens in its outputs if the token 
+identifier is equal to the identifier of the first input box of the transaction.
+As the box identifier is cryptographically unique, there's no chance to have the second token with the same identifier while the hash function being used 
+is collision-resistant. 
+
+In order to verify the authenticity of a token, the unique identifier of a token is needed to check.
+
+## Recommended approach for applications showing tokens to end-users
+
+### Verified tokens 
+Applications should add a verification sign next to a token which is listed in the following [genuine tokens list](#genuine-tokens).
+
+Proposed verification sign: [Material icons verified](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Averified%3A)
+
+### Suspicious tokens
+In order to protect end-users for confusion, it is decided for some tokens in the [genuine tokens list](#genuine-tokens) that the verbose name should be 
+unique and should not be used by other tokens. 
+This is not enforced by Ergo protocol, so applications should check if a token uses a unique name from the list and add a warning sign when needed.
+
+Proposed warning sign: [Material icons report](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Areport%3A)
+
+### Blocked tokens
+Applications should show a warning sign next to tokens identified to be harmful or impersonating other tokens.
+
+Proposed warning sign: [Material icons dangerous](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Adangerous%3A)
+
+## Token lists
+
+### Genuine tokens
+
+| Verbose name       | Token id                                        | Unique name |
+| ------------------ |:-----------------------------------------------:| -----------:|
+| SigUSD             |                                                 | yes         |
+| SigRSV             |                                                 | yes         |
```

Comment:
Not sure the `Unique name` column is necessary.
Also please put the actual ids for the tokens in the table.

Reply:
Do you think unique name is not necessary because the name for tokens on the list should always be unique, or because you think warning about an impersonated token name is not necessary?

Reply:
Added the IDs